src/lib.rs

```diff
@@ -410,6 +412,40 @@ impl Contract {
     pub fn get_community(&self, handle: String) -> Option<Community> {
         self.communities.get(&handle)
     }
+
+    pub fn add_featured_communities(&mut self, handles: Vec<String>) {
```

Comment:
```suggestion
    pub fn set_featured_communities(&mut self, handles: Vec<String>) {
```
Since it's overwrite instead of insert

Reply:
I have made the changes for the function name.